src/site/content/en/blog/codelab-building-a-sidenav-component/index.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: codelab
+title: "Codelab: Building a Sidenav component"
+authors:
+  - adamargyle
+description: |
+  Learn how to build a responsive slide out side navigation layout component.
+date: 2020-11-25
+hero: hero.jpg
+thumbnail: thumb.jpg
+glitch: gui-challenges-sidenav-lab
+glitch_path: app/index.html
+related_post: building-a-sidenav-component
+tags:
+  - css
+  - dom
+  - javascript
+  - layout
+  - mobile
+  - ux
+---
+
+This codelab teaches you how to build a responsive slide out side navigation layout component
+on the web. We'll build the component as we go, starting with HTML, then CSS,
+then JavaScript.
+
+Check out my blog post [Building a Sidenav component](/building-a-sidenav-component)
+to learn about the CSS tools chosen for building this component.
```

Comment:
As mentioned in the blog post, I worry "tools" might be a bit misleading for some readers.